html/changelog.html

```diff
@@ -516,12 +516,6 @@ <h3 class="author">Putnam3145 updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">fixes a major source of lag</li>
 			</ul>
-
-			<h2 class="date">14 July 2021</h2>
-			<h3 class="author">Putnam3145 updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="refactor">Acid is now a component</li>
-			</ul>
```

Comment:
Don't touch the changelog files, the bot is supposed to do that.

Reply:
sometimes this just kind of happens randomly on compile